packages/common/src/vscode-webui-bridge/webview.ts

```diff
@@ -80,6 +80,18 @@ export interface VSCodeHostApi {
     }[]
   >;
 
+  listAutoCompleteCandidates(
+    query?: string,
+    options?: {
+      symbolLimit?: number;
+    },
```

Comment:
```suggestion
  limit?: number;
```

Just use a global limit